convert2rhel/actions/system_checks/package_updates.py

```diff
@@ -55,12 +55,12 @@ def run(self):
         reposdir = get_hardcoded_repofiles_dir()
 
         if reposdir and not system_info.has_internet_access:
-            logger.warning("Skipping the check as no internet connection has been detected.")
+            logger.warning("Do not perform the check as no internet connection has been detected.")
             self.add_message(
                 level="WARNING",
                 id="PACKAGE_UPDATES_CHECK_SKIP_NO_INTERNET",
-                title="Skipping the package updates check",
-                description="Skipping the check as no internet connection has been detected.",
+                title="Do not perform the package updates check",
+                description="Do not perform the check as no internet connection has been detected.",
```

Comment:
It seems better to say `Did not perform` instead for some of these files

Reply:
I'll switch to "Did not perform the check"